dom.bs

```diff
@@ -6774,15 +6832,23 @@ are:
   </ol>
  </li>
 
- <li><p>If <a>this</a> is a <a for=Element>shadow host</a>, then <a>throw</a> an
- "{{NotSupportedError!!exception}}" {{DOMException}}.
+ <li><p>If <var>shadow host</var> has a non-null <a for=/>shadow root</a>, then:
```

Comment:
Why can this not use the original language of "If _shadow host_ is a shadow host"?

Reply:
Done.